aws/resources/templates/invitation_template.html

```diff
@@ -1,12 +1,80 @@
-<!doctype html>
-<html>
-    <head>
-        <meta charset="utf-8">
-        <title>Wedding Invitation for the Flying Js!</title>
-        <meta name="description" content="Invitation to the wedding of Jenny Rosenberger and Jesse Kelly.">
-        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    </head>
-    <body>
-        <p>You're invited!</p>
-    </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1"/>
+  <title>Invitation | Flying J Wedding</title>
+  <link rel="icon" 
+      type="image/png" 
+      href="logo-favicon.png">
+
+  <!-- CSS  -->
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  
+<link href="https://fonts.googleapis.com/css?family=Cormorant" rel="stylesheet">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css">
+  <link href="../public/css/template-style.css" type="text/css" rel="stylesheet" media="screen,projection"/>
+</head>
+<body>
+  <nav class="taupe" role="navigation">
+    <div class="nav-wrapper  taupe container">
+      <a id="logo-container" href="#" class="brand-logo"><img src="public/images/Flying-J-Stand-Alone_Logo.png" alt="Flying J Wedding"></a>
+      <ul class="right hide-on-med-and-down">
+        <li><a href="index.html#home">Home</a></li>
+        <li><a href="index.html#travel">Travel</a></li>
+        <li><a href="index.html#gifts">Gifts</a></li>
+      </ul>
+
+      <ul id="nav-mobile" class="taupe side-nav">
+        <li><a href="index.html#home">Home</a></li>
+        <li><a href="index.html#travel">Travel</a></li>
+        <li><a href="index.html#gifts">Gifts</a></li>
+      </ul>
+      <a href="#" data-activates="nav-mobile" class="button-collapse"><i class="material-icons">menu</i></a>
+    </div>
+  </nav>
+<main class="light-tan">
+  <section">
+    <div class="section container">
+      <div class="row">
+        <div class="col s12 m8 push-m2 l6 push-l3">
+          <img class="responsive-img z-depth-4 center" src="../public/images/wedding invitation-03.png" alt="invitation">
```

Comment:
Remove `public`

Reply:
Watch out for whitespace, `wedding invitation-03.png` may work on your local machine, but whitespace needs to be URL-encoded (or just change the filename so that there isn't a space in it)